.github/workflows/master.yml

```diff
@@ -243,9 +249,9 @@ jobs:
     timeout-minutes: 120
     strategy:
       matrix:
-        group: [1, 2, 3]
+        group: [1, 2]
         include:
-          - splits: 3
+          - splits: 2
```

Comment:
Chose 2. The number of jobs is amplified by the number of people working in this repo. We could choose 3 if that's feasible.